jenkins-build-pack-lambda/Jenkinsfile

```diff
@@ -135,7 +135,7 @@ node()  {
 	}
 
 	dir(repo_name){
-		loadServerlessConfig(runtime, isEventSchdld, isScheduleEnabled, isEc2EventEnabled, isS3EventEnabled, isStreamEnabled, isDynamoDbEnabled)
+		loadServerlessConfig(runtime, isEventSchdld, isScheduleEnabled, isEc2EventEnabled, isS3EventEnabled, isStreamEnabled, isDynamoDbEnabled, service)
```

Comment:
remove runtime and service from method signature and pass only config. And take runtime and service from config in side method

Reply:
Done